src/modules/item/Item.tsx

```diff
@@ -368,6 +357,7 @@ const Item = ({
       return (
         <H5PItem
```

Comment:
Doesn't this need the item? Or h5p cannot collapse? Same for Etherpad.

Reply:
@MalinSvenberg added the support for collapsing an H5P item in graasp-ui [v2.4.0](https://github.com/graasp/graasp-ui/releases/tag/v2.4.0). Etherpad items are not supported currently.

In the api design we choose to only add the `itemName` prop instead of passing the whole `item`. We might consider uniformizing the API later if it makes sens.